docs/content/concepts/registering-kubernetes-clusters-using-syncer.md

```diff
@@ -61,53 +61,54 @@ To register a Kubernetes cluster with kcp, you have to install a special compone
     deployment.apps/kcp-syncer-kind-1owee1ci created
     ```
 
-    and it will create a `kcp-syncer` deployment:
+    and it will create a `kcp-syncer-<synctarget name>-xx` deployment:
 
     ```sh
     $ KUBECONFIG=<pcluster-config> kubectl -n kcp-syncer-kind-1owee1ci get deployments
-    NAME     READY   UP-TO-DATE   AVAILABLE   AGE
-    kcp-syncer   1/1     1            1           13m
+    NAME     READY                    UP-TO-DATE   AVAILABLE   AGE
+    kcp-syncer-<synctarget name>-xx   1/1     1            1           13m
     ```
 
 1. Wait for the kcp sync target to go ready:
 
     ```bash
     kubectl wait --for=condition=Ready synctarget/<mycluster>
     ```
-### Select resources to sync.
 
-Syncer will by default use the `kubernetes` APIExport in `root:compute` workspace and sync `deployments/services/ingresses`
-to the physical cluster. The related API schemas of the physical cluster should be comptible with kubernetes 1.24. User can
-select to sync other resources in physical clusters or from other APIExports on kcp server.
+### Select resources to synchronize.
+
+Syncer will by default use the `kubernetes` APIExport in `root:compute` workspace and synchronize `deployments/services/ingresses/pods`
```

Comment:
IIRC `pod` is only `upsync`ed. Also, there's a PR open to change from `kubernetes` https://github.com/kcp-dev/kcp/pull/2877 @qiujian16 can you update the doc as well please